pandoc/filter-espd.py

```diff
@@ -0,0 +1,98 @@
+#! /usr/bin/env python
+# encoding: utf-8
+from panflute import Link, run_filter, Para, Str, Header, MetaInlines, Space, debug, Div, HorizontalRule, \
+    BulletList, RawBlock, Span, Emph, Strong, Image, Block, Inline, ListContainer
+
+version = None
+
+pb = RawBlock(u"<w:p><w:r><w:br w:type=\"page\" /></w:r></w:p>", format=u"openxml")
+
+
+def action(elem, doc):
+    if isinstance(elem, Image):
+        # debug(elem.content)
+        if isinstance(elem.content, ListContainer):
+            attributes={'custom-style': 'image'}
+            # if hasattr(elem.parent, 'attributes'):
+            # debug(elem.parent)
+
+        return elem
+    if isinstance(elem, MetaInlines):
+        """
+        Title page header
+        """
+        global version
```

Comment:
Global variables are a bad practice in programming (because magical effects occur).

Reply:
fixed